source/developers-guide/shopware-5-cli-commands/index.md

```diff
@@ -65,3 +65,27 @@ Loads snippets from the `snippets` folder of your Shopware installation and crea
 - `include-plugins`: If set, active plugin snippets will also be exported.
 
 - `update`: By default, the generated SQL script only performs inserts. If the `update` option is provided, it will also handle update scenarios when duplicates are found. If the existing database snippet has `dirty` = 0, the value will be overwritten. If `dirty` is 1, it's not changed. Please note that, for a large number of snippets, enabling update support will make the SQL statements significantly slower to execute upon importing.
+
+## Themes
+
+### sw:theme:create
```

Comment:
That section just duplicates what is already documented in the command itself. Please see my comment in the conversation view.